src/dictionary/ui/search/DictionaryLineGroup.tsx

```diff
@@ -0,0 +1,159 @@
+import { isTextLine } from 'transliteration/domain/type-guards'
+import ManuscriptPopOver from 'corpus/ui/ManuscriptPopover'
+import { parallelLinePrefix } from 'transliteration/domain/parallel-line'
+import DictionaryLineVariant from 'dictionary/ui/search/DictionaryLineVariant'
+
+function createCorpusChapterUrl(
+  textId: TextId,
+  stage: string,
+  name: string
+): string {
+  const urlParts = [
+    textId.genre,
+    textId.category,
+    textId.index,
+    stageToAbbreviation(stage),
+    name,
+  ]
+
+  return `https://www.ebl.lmu.de/corpus/${urlParts
+    .map(encodeURIComponent)
+    .join('/')}`
+}
+
+function LemmaLineHeader({
+  lemmaLine,
+}: {
+  lemmaLine: DictionaryLineDisplay
+}): JSX.Element {
+  return (
+    <tr>
+      <th scope="col" colSpan={2} className="lines-with-lemma__header">
+        <span className="lines-with-lemma__textname">
+          <InlineMarkdown source={lemmaLine.textName} />
+        </span>
+        &nbsp;
+        <span>
+          (
+          <a
+            target="_blank"
+            rel="noopener noreferrer"
+            href={createCorpusChapterUrl(
+              lemmaLine.textId,
+              lemmaLine.stage,
+              lemmaLine.chapterName
+            )}
+          >
+            {lemmaLine.textId.genre} {textIdToString(lemmaLine.textId)}
+          </a>
+          )
+        </span>
+        &nbsp;
+        <span>{lemmaLine.chapterName}</span>
+      </th>
+    </tr>
+  )
+}
+
+function DictionaryManuscriptLines({
+  variants,
+  maxColumns,
+  lemmaId,
+}: {
+  variants: readonly LineVariantDetails[]
+  maxColumns: number
+  lemmaId: string
+}): JSX.Element {
+  return (
+    <>
+      {variants.map((variant) =>
+        variant.manuscripts.map((manuscript, index) => {
+          return (
+            <tr key={index} className="lines-with-lemma__manuscript-line">
+              <td>
+                {manuscript.isParallelText && parallelLinePrefix}
+                <ManuscriptPopOver manuscript={manuscript} />
+              </td>
+              {isTextLine(manuscript.line) ? (
+                <td>
+                  <table>
+                    <tbody>
+                      <tr>
+                        <LineColumns
+                          columns={manuscript.line.columns}
+                          maxColumns={maxColumns}
+                          isInLineGroup={false}
+                          highlightLemma={lemmaId}
+                        />
+                      </tr>
+                    </tbody>
+                  </table>
+                </td>
+              ) : (
+                <td colSpan={maxColumns}></td>
+              )}
+            </tr>
+          )
+        })
+      )}
+    </>
+  )
+}
+
+export default function DictionaryLineGroup({
+  lines,
+  lemmaId,
+}: {
+  lines: DictionaryLineDisplay[]
+  lemmaId: string
+}): JSX.Element {
+  const columns = lines.map((dictionaryLine) =>
+    createColumns(dictionaryLine.line.variants[0].reconstruction)
```

Comment:
Is this right?

Reply:
@khoidt okay, I found where this comes from: [ChapterView.tsx](https://github.com/ElectronicBabylonianLiterature/ebl-frontend/blob/cc5bd57a6c3b0cc478e0119db1030f00092c1a87/src/corpus/ui/ChapterView.tsx#L88). I guess I'll just leave it as is then?